src/components/pages/AppMarketplace/components/FavoriteSection/index.tsx

```diff
@@ -56,7 +56,7 @@ export default function FavoriteSection() {
         <CarouselBox
           title={t('content.appstore.favoriteSection.myFavorite')}
           itemWidth={266}
```

Comment:
I have tested it and it looks fine to me, however.
there is small edge showing up for the previous items, so if you set 270px width, it will fix that as well.